src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -234,6 +254,213 @@ def _build_for_tgi_jumpstart(self):
+                        p90,
+                        avg_tokens_per_second,
+                        throughput_per_second,
+                        standard_deviation,
+                    ]
+                    if _more_performant(best_tuned_combination, tuned_configuration):
+                        best_tuned_combination = tuned_configuration
+            except LocalDeepPingException as e:
+                logger.warning(
+                    "Deployment unsuccessful with %s: %s. SAGEMAKER_MODEL_SERVER_WORKERS: %s"
+                    "Failed to invoke the model server: %s",
+                    num_shard_env_var_name,
+                    tensor_parallel_degree,
+                    sagemaker_model_server_workers,
+                    str(e),
+                )
+                break
+            except LocalModelOutOfMemoryException as e:
+                logger.warning(
+                    "Deployment unsuccessful with %s: %s. SAGEMAKER_MODEL_SERVER_WORKERS: %s. "
+                    "Out of memory when loading the model: %s",
+                    num_shard_env_var_name,
+                    tensor_parallel_degree,
+                    sagemaker_model_server_workers,
+                    str(e),
+                )
+                break
+            except LocalModelInvocationException as e:
+                logger.warning(
+                    "Deployment unsuccessful with %s: %s. SAGEMAKER_MODEL_SERVER_WORKERS: %s. "
+                    "Failed to invoke the model server: %s"
+                    "Please check that model server configurations are as expected "
+                    "(Ex. serialization, deserialization, content_type, accept).",
+                    num_shard_env_var_name,
+                    tensor_parallel_degree,
+                    sagemaker_model_server_workers,
+                    str(e),
+                )
+                break
+            except LocalModelLoadException as e:
+                logger.warning(
+                    "Deployment unsuccessful with %s: %s. SAGEMAKER_MODEL_SERVER_WORKERS: %s. "
+                    "Failed to load the model: %s.",
+                    num_shard_env_var_name,
+                    tensor_parallel_degree,
+                    sagemaker_model_server_workers,
+                    str(e),
+                )
+                break
+            except SkipTuningComboException as e:
+                logger.warning(
+                    "Deployment with %s: %s. SAGEMAKER_MODEL_SERVER_WORKERS: %s. "
+                    "was expected to be successful. However failed with: %s. "
+                    "Trying next combination.",
+                    num_shard_env_var_name,
+                    tensor_parallel_degree,
+                    sagemaker_model_server_workers,
+                    str(e),
+                )
+                break
+            except Exception:  # pylint: disable=W0703
+                logger.exception(
```

Comment:
Keep all the log levels as logger.exception here

Reply:
This is similar to existing ones https://github.com/aws/sagemaker-python-sdk/blob/959cb0f557d1dd9a8a3d99dcb72df46665ea75c3/src/sagemaker/serve/builder/djl_builder.py#L391-L436

Reply:
Ack so keep similar experience. lgtm